### DISCUSSION TITLE
Rejected from GitHub Education Pack | but I'm overseas

### DISCUSSION THREAD

**Discussion by ErikPDev** [community/community#60250] [GitHub Education] [answered]
### Hi everyone,

Hi there!
I recently applied for the GitHub Education Pack, but slight issue-
I'm not in the same country as the school since I'm currently overseas for vacation.

Rejection Reasons:
- The image you used appears to be edited. Manipulated images may not be used. Please try a different image.
- The school you selected does not appear to have a campus location in your country. Your school-provided academic affiliation document must indicate virtual learning if you are remote. If you believe that the school you have applied for has incorrect information that affects your current application e.g. incorrect email domain, please select ‘Other’ option when submitting a GitHub Education support ticket.
- You appear not to be near any campus location for the school you have selected. Your school-provided academic affiliation document must indicate virtual learning if you are remote. Please enable the application to discover your location from the browser pop-up to provide an accurate location. If you are using a VPN, please disable it when you reapply. If you believe that the school you have applied for has incorrect information that affects your current application e.g. incorrect campus location, please select ‘Other’ option when submitting a GitHub Education support ticket.

I'm not quite sure why the image appears to be edited since I screenshotted my transcript from the school's portal

Any help will be incredible! Thanks in regards!

**Comment by samus-aran** [comment, marked as answer]
Hi @ErikPDev 👋 

Try not to use a file picker, but submit a live photo of the evidence. If possible, we encourage students to apply whilst on campus. This will assist us in confirming your student status.
